### PR TITLE
Pull request for rework-find-measures

### PR DESCRIPTION
**Pull request by jd**
storage: move MeasureQuery in REST API
storage: remove find_measures
storage: make find_measures batched

### CODE REVIEW

File: gnocchi/rest/api.py [hunk 1]
```diff
@@ -1484,35 +1579,49 @@ def post(self, metric_id, start=None, stop=None, aggregation='mean',
...
                 reverse=True)
 
-        results = {}
+        metrics_and_aggregations = collections.defaultdict(list)
+
+        for metric in metrics:
+            if granularity is None:
+                granularity = sorted((
+                    d.granularity
+                    for d in metric.archive_policy.definition),
```
Comment (chungg) [inline gnocchi/rest/api.py]: i imagine we can just check aggregations here while we're looping through the list.

Comment (jd) [reply gnocchi/rest/api.py]: I don't understand what you want to check here?

Comment (jd) [reply gnocchi/rest/api.py]: Yeah I don't think it matters at that point :)

**Comment by jd** [comment]
Pull-request updated, HEAD is now 10e6237700aa45114620248192907446026efcd4

**Pull request merged**